### ISSUE TITLE
Rounding percentages

### ISSUE THREAD

**Issue by sconnelley** [cityenergyproject/seattle#111]
The scorecard fuel use chart shows rounded percentages, which can sometimes lead to sums not equal to 100.  Need client to decide what they want to do about this.  If we round to the nearest tenth then this becomes less of a problem.

```
#seattle/2016?filters[0][field]=site_eui&filters[0][max]=15&layer=site_eui&sort=site_eui&order=desc&lat=47.62294&lng=-122.33877&zoom=12&building=659&report_active=true
```

![image](https://user-images.githubusercontent.com/565486/29568785-4c5718dc-8706-11e7-9b12-2da9418fc1de.png)

**Comment by sconnelley** [comment]
Using something similar to the [Largest Remainder Method](https://en.wikipedia.org/wiki/Largest_remainder_method)

![image](https://user-images.githubusercontent.com/565486/30933077-7013634a-a397-11e7-838c-a234300fa2f4.png)

**Comment by sconnelley** [comment]
fixed in a481197b8fefe95151c63220c5eb771caa4f23e0
